src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -689,13 +696,18 @@ private function rowToArray(
             return false;
         }
 
-        if ($document instanceof SecurityBehavior && $permission) {
+        if ($document instanceof SecurityBehavior && $this->security && $permission && $webspace->hasWebsiteSecurity()) {
```

Comment:
`$webspace` key could be null, think we should here also do:

```suggestion
        if ($document instanceof SecurityBehavior && $this->security && $permission && $webspace && $webspace->hasWebsiteSecurity()) {
```

Else we would need to throw some kind of exception for an unexpected call.